phpunit/class-rest-blocks-controller-test.php

```diff
@@ -8,7 +8,7 @@
 /**
  * Tests for WP_REST_Blocks_Controller.
  */
-class REST_Blocks_Controller_Test extends WP_Test_REST_Controller_Testcase {
+class REST_Blocks_Controller_Test extends WP_UnitTestCase {
```

Comment:
This change might make someone somewhere upset, but implementing 9 abstract methods when the controller implements only `check_read_permission` is excessive, and would hint more at unreliability of tests for the extended posts controller.

Reply:
> implementing 9 abstract methods when the controller implements only `check_read_permission` is excessive, and would hint more at unreliability of tests for the extended posts controller.

Yeah, it's annoying. We can improve upon it at some point.